### ISSUE TITLE
CreateTableDefaultPermissions argument is not supported for aws_glue_catalog_database creation

### ISSUE THREAD

**Issue by peramnaveen29** [hashicorp/terraform-provider-aws#14411]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
Terraform docs says only the following arguments are supported:

name - (Required) The name of the database.
catalog_id - (Optional) ID of the Glue Catalog to create the database in. If omitted, this defaults to the AWS Account ID.
description - (Optional) Description of the database.
location_uri - (Optional) The location of the database (for example, an HDFS path).
parameters - (Optional) A list of key-value pairs that define parameters and properties of the database.

### New or Affected Resource(s)

* aws_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #9700

**Comment by peramnaveen29** [comment]
@ewbankkit Is this gonna be added along with the enhancement #9700 as mentioned in [August - October 2020 Roadmap](https://github.com/terraform-providers/terraform-provider-aws/blob/master/ROADMAP.md#aws-lake-formation) ?

**Comment by ewbankkit** [comment]
@peramnaveen29 I have added the Lake Formation service label to this issue so that it can be considered as part of the overall Lake Formation support story.
Thanks.

**Comment by YakDriver** [comment]
This functionality is available with Lake Formation (see #13250 and #13396) and available in 3.22. I'm going to close this but if you continue to have issues not addressed above, please open a new issue. Thanks for your interest!
